src/telegram.js

```diff
@@ -845,6 +845,165 @@ class TelegramBot extends EventEmitter {
+   * @param {Object} [options] Additional Telegram query options
+   * @returns {Promise}
+   * @see https://core.telegram.org/bots/api#deletechatphoto
+   */
+  deleteChatPhoto(chatId, form = {}) {
+    form.chat_id = chatId;
+    return this._request('deleteChatPhoto', { form });
+  }
+
+  /**
+   * Use this method to change the title of a chat. Titles can't be changed for private chats.
+   * The bot must be an administrator in the chat for this to work and must have the appropriate admin rights.
+   * Returns True on success.
+   *
+   * Note: In regular groups (non-supergroups), this method will only work if the ‘All Members Are Admins’
+   * setting is off in the target group.
+   *
+   * @param {Number|String} chatId  Unique identifier for the message recipient
+   * @param {String} title New chat title, 1-255 characters
+   * @param {Object} [options] Additional Telegram query options
+   * @returns {Promise}
+   * @see https://core.telegram.org/bots/api#setchattitle
+   */
+  setChatTitle(chatId, title, form = {}) {
+    form.chat_id = chatId;
+    form.title = title;
+    return this._request('setChatTitle', { form })
+  }
+
+  /**
+   * Use this method to change the description of a supergroup or a channel.
+   * The bot must be an administrator in the chat for this to work and must have the appropriate admin rights.
+   * Returns True on success.
+   *
+   * @param {Number|String} chatId  Unique identifier for the message recipient
+   * @param {String} description New chat title, 1-255 characters
+   * @param {Object} [options] Additional Telegram query options
+   * @returns {Promise}
+   * @see https://core.telegram.org/bots/api#setchatdescription
+   */
+  setChatDescription(chatId, description, form = {}) {
+    form.chat_id = chatId;
+    form.description = description;
+    return this._request('setChatDescription', { form })
+  }
+
+  /**
+   * Use this method to pin a message in a supergroup.
+   * The bot must be an administrator in the chat for this to work and must have the appropriate admin rights.
+   * Returns True on success.
+   *
+   * @param {Number|String} chatId  Unique identifier for the message recipient
+   * @param {String} messageId Identifier of a message to pin
+   * @param {Object} [options] Additional Telegram query options
+   * @returns {Promise}
+   * @see https://core.telegram.org/bots/api#pinchatmessage
+   */
+  pinChatMessage(chatId, messageId, form = {}) {
+    form.chat_id = chatId;
+    form.message_id = messageId;
+    return this._request('pinChatMessage', { form });
+  }
+
+  /**
+   * Use this method to unpin a message in a supergroup chat.
+   * The bot must be an administrator in the chat for this to work and must have the appropriate admin rights.
+   * Returns True on success.
+   *
+   * @param {Number|String} chatId  Unique identifier for the message recipient
+   * @param {Object} [options] Additional Telegram query options
```

Comment:
No additional parameters are required for this method too. See [here](https://core.telegram.org/bots/api#unpinchatmessage).